src/modules/Hosts/Hosts/Helpers/NativeEventWaiter.cs

```diff
@@ -14,7 +15,7 @@ public static class NativeEventWaiter
         public static void WaitForEventLoop(string eventName, Action callback)
         {
             var dispatcherQueue = DispatcherQueue.GetForCurrentThread();
-            new Thread(() =>
```

Comment:
Since this is long-running, wouldn't it be better to set [`IsBackground`](https://learn.microsoft.com/en-us/dotnet/api/system.threading.thread.isbackground?view=net-8.0) instead?

Reply:
good callout! updated to use background threads

Reply:
Ok